docs/en/observability/profiling-tag-data-query.asciidoc

```diff
@@ -8,7 +8,7 @@ The only config setting you may want to change is `project-id` (default value is
 
 The `-project-id` flag, or the `project-id` key in the host-agent configuration file, splits profiling data into logical groups that you control.
 
-You can assign any non-zero, unsigned integer <= 4095 to a host-agent deployment you control. In Kibana, the KQL field `profiling.project.id` is mapped to `project-id` and you can use it to split or filter data.
+You can assign any non-zero, unsigned integer <= 4095 to a host-agent deployment you control. In Kibana, the KQL field `service.name` is mapped to `project-id` and you can use it to split or filter data.

```

Comment:
We should probably leave this as-is? the new field is going to be visible as `profiling.project.id` in the UI?

Reply:
Ah, I missed that one. Good catch!

Reply:
Addressed in https://github.com/elastic/observability-docs/pull/3108/commits/99f8948f4465be42446869af6b43b7947e6784cc.